src/Blocks/BlockCrops.h

```diff
@@ -21,6 +21,24 @@ class cBlockCropsHandler:
 
 private:
 
+	static char FortuneBinomialRandom(char a_Min, char a_Rolls)
```

Comment:
Since this function is only used here, I would rather it took 3 argumentsL min, baseRolls, and fortuneLevel; and do the addition internally. 

It would also be better named something like CalculateDropCount - it feels slightly wrong for BlockCrops.h to have the so-named source of truth on fortune calculations.